### ISSUE TITLE
button:(Platform) Text alternative needs to be provided for the icons button 

### ISSUE THREAD

**Issue by sKudum** [SAP/fundamental-ngx#2965]
#### Is this a bug, enhancement, or feature request?
bug
#### Briefly describe your proposal.
There is no text alternative provided for the icons button as highlighted.
Also, Speech output is made as ‘Button’, No label associated with the UI.
<img width="557" alt="Screen Shot 2020-08-03 at 9 51 46 AM" src="https://user-images.githubusercontent.com/53487468/89145648-09292900-d56f-11ea-9e74-a843c8e42415.png">

Meaning label/text alternative shall be provided on accessing the non-text UI elements within the application.
Also, it should be associated/conveyed by the screen reader.

#### Which versions of Angular and Fundamental Library for Angular are affected? (If this is a feature request, use current version.)
9

#### If this is a bug, please provide steps for reproducing it.

1).Activate screen readers and focus on icon button. it should read out thee button details.

**Comment by sKudum** [comment]
closing it as it is addressed and merged in PR#2995
